bowling/inc/frameParser.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <string>
+#include <vector>
+
+class FrameParser {
+public:
+    FrameParser(const std::string& gameString);  // thinking about explicit...
+
+    std::string getGameString() const { return gameString_; }
+    std::vector<size_t> getFrames() const { return frames_; }
+
+    void parse();
```

Comment:
can be turn into  static function
```suggestion
    static std::vector<size_t> parse(const std::string& gamePoints){
        //parsingString into frames vector
        return frames;
    }
```

Reply:
This is vip, we will see how it goes.